### PR TITLE
[Tag]: fixed size of layouts

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: uui/components/widgets/Tag.module.scss [hunk 1]
```diff
@@ -67,75 +67,74 @@
...
+        padding: 2px;
 
         :global(.uui-caption), :global(.uui-input) {
-            padding-top: 2px;
-            padding-bottom: 2px;
-            line-height: 18px;
-            font-size: 14px;
+            padding: 3px 3px;
+            line-height: 12px;
+            font-size: 12px;
```
Comment (siarhei-epam) [inline uui/components/widgets/Tag.module.scss:91]: It's danger

File: uui/components/widgets/Tag.module.scss [hunk 2]
```diff
@@ -67,75 +67,74 @@
...
             margin-right: 3px;
         }
 
         :global(.uui-caption), :global(.uui-input) {
-            padding: 2px 3px;
+            padding: 0 3px;
             line-height: 12px;
             font-size: 12px;
         }
     }
```
Comment (siarhei-epam) [inline uui/components/widgets/Tag.module.scss:81]: Why you change vertical layout in all sizes?

File: uui/components/widgets/Tag.module.scss [hunk 3]
```diff
@@ -67,75 +67,74 @@
...
 
     &.size-24 {
-        @include button-horizontal-layout(24px, 1px);
+        min-width: 24px;
         min-height: 24px;
+        padding: 2px;
 
         :global(.uui-caption), :global(.uui-input) {
```
Comment (AlekseyManetov) [inline uui/components/widgets/Tag.module.scss:88]: why do we have `:global(.uui-input)` in tag related css?

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui/components/widgets/Tag.module.scss (hunk 1 above)
Comment (vik753) [reply uui/components/widgets/Tag.module.scss]: fixed

File: uui/components/widgets/Tag.module.scss (hunk 2 above)
Comment (vik753) [reply uui/components/widgets/Tag.module.scss]: checked and set optimal sizes

File: uui/components/widgets/Tag.module.scss (hunk 3 above)
Comment (vik753) [reply uui/components/widgets/Tag.module.scss]: fixed

**Review by siarhei-epam — APPROVED** [review]

**Pull request merged**